src/scenes/NftDetailPage/NavigationHandle.tsx

```diff
@@ -20,63 +20,94 @@ const HOVER_TEXT = new Map<number, string>([
 
 type Props = {
   direction: Directions;
+  username: string;
+  collectionId: string;
   nftId: string;
 };
 
-function NavigationHandle({ direction, nftId }: Props) {
+function NavigationHandle({ direction, username, collectionId, nftId }: Props) {
   const arrow = useMemo(() => ARROWS.get(direction) ?? '', [direction]);
 
   const hoverText = useMemo(() => HOVER_TEXT.get(direction) ?? '', [direction]);
 
-  const { push } = useRouter();
+  const { replace } = useRouter();
   const handleOnClick = useCallback(() => {
     // TODO(Terence): Figure out how to get this state across since Next doesn't support navigation state
     // void push(nftId, { state: { collection: [] } });
-    void push({ pathname: nftId });
-  }, [nftId, push]);
+    // void push({ pathname: nftId });
+    void replace(`/${username}/${collectionId}/${nftId}`);
+  }, [nftId, collectionId, username, replace]);
 
   return (
-    <StyledNavigationHandle onClick={handleOnClick}>
-      {/* <StyledGradient /> */}
+    <StyledNavigationHandle onClick={handleOnClick} direction={direction}>
       <StyledTextWrapper direction={direction}>
         {arrow}
-        <Spacer width={8} />
-        <StyledHoverText color={colors.gray50} caps>
-          {hoverText}
+        <Spacer width={3} />
+        <StyledHoverText>
+          <ActionText>{hoverText}</ActionText>
         </StyledHoverText>
       </StyledTextWrapper>
     </StyledNavigationHandle>
   );
 }
 
-const StyledTextWrapper = styled.div<{ direction: Directions }>`
+const StyledTextWrapper = styled(ActionText)<{ direction: Directions }>`
```

Comment:
We should avoid turning `StyledTextWrapper` into a p tag (`ActionText`) if we can because `StyledTextWrapper` has children that are block elements (the hover text + spacer divs).

Technically p tags shouldn't contain block elements like `div`s or other `p`s cus its invalid html:
https://stackoverflow.com/a/8696078


if the intention was to style the arrow, maybe we could do this instead: 
`<ActionText>{arrow}</ActionText>`